pipestat_example/looper_config_pipestat.yaml

```diff
@@ -0,0 +1,32 @@
+pep_config: ./project/project_config_pipestat.yaml # pephub registry path or local path
+output_dir: "./results"
+pipeline_interfaces:
+  sample:  ./pipeline/pipeline_interface.yaml
+  project: ./pipeline/pipeline_interface_project.yaml
+pipestat:
+  project:
+    project_name: test
+    sample_name: frog_1
```

Comment:
it doesn't make sense to me to have a sample_name in a looper config file like this.